draft-ietf-acme-acme.md

```diff
@@ -1944,11 +1932,13 @@ for the domain in question.
 
 The path at which the resource is provisioned is comprised of the fixed prefix
 ".well-known/acme-challenge/", followed by the "token" value in the challenge.
-The value of the resource MUST be the ASCII representation of the key
```

Comment:
Although I guess that doesn't have the normative language... I'd be happy to keep this line (minus the ASCII reference) and delete that line from the preceding parahraph as a compromise?